src/Agoda.Analyzers/AgodaCustom/AG0006NotOveruseReflection.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Agoda.Analyzers.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+
+namespace Agoda.Analyzers.AgodaCustom
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class AG0006NotOveruseReflection : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "AG0006";
```

Comment:
Manu has already used this ID in his rule. How can we stop these conflicts in the future, because as more people write rules, they are going to become common.

Reply:
What I have done is, looked at the open pull requests, decided on rule ID, created a pull request immediately with the ID in the title. Then we can solve conflicts in an FCFS manner.